app/models/search/tag_query.rb

```diff
@@ -103,6 +108,19 @@ def suggested_character_filter
     terms_filter(:pre_character_ids, options[:pre_character_ids]) if options[:pre_character_ids]
   end
 
+  def to_wrangle_filter
```

Comment:
I'd split this filter and generally try to keep filters mostly one-to-one with the columns we indexed. I think that makes queries easier to read.

We already have `unwrangleable_filter`, so you can add an `in_use_filter` for "used OR canonical", and a simple `unwrangled_filter` that's just a boolean check matching the indexed column. Then you can use all 3 when setting up the query in the controller.

Simpler filters means you can avoid hard `raise` errors for unexpected combinations as well.